src/clipper.js

```diff
@@ -106,7 +106,11 @@ export default class Clipper {
     //TODO: subscribe to file events to update dog, calc and other parameters
   }
 
-  activeAuctions() {
+  async activeAuctions() {
```

Comment:
This function is called with an `await`, so this should be an `async` function.  I needed this for the debug mode.